### ISSUE TITLE
XML is not well-formed

### ISSUE THREAD

**Issue by agentram** [CycloneDX/cyclonedx-python#139]
Non-XML characters may get into a BOM file.
```shell
$ pip3 freeze | grep cyclonedx
cyclonedx-bom==0.4.2

$ cat requirements.txt
fuzzywuzzy==0.15.0

$ cyclonedx-py
Input file: requirements.txt
Output BOM: bom.xml
JSON output: False
Package info url: https://pypi.org/pypi/{package_name}/{package_version}/json
Generating CycloneDX BOM
Validating BOM
Traceback (most recent call last):
  File "/usr/local/bin/cyclonedx-py", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/site-packages/cyclonedx/client.py", line 85, in main
    validate_bom(args)
  File "/usr/local/lib/python3.8/site-packages/cyclonedx/client.py", line 70, in validate_bom
    if validator.is_valid(args.output_file, args.json):
  File "/usr/local/lib/python3.8/site-packages/cyclonedx/bom/validator.py", line 39, in is_valid
    return xml_bom_schema.is_valid(output_filename)
  File "/usr/local/lib/python3.8/site-packages/xmlschema/validators/schema.py", line 1435, in is_valid
    error = next(self.iter_errors(source, path, schema_path, use_defaults, namespaces), None)
  File "/usr/local/lib/python3.8/site-packages/xmlschema/validators/schema.py", line 1456, in iter_errors
    source = XMLResource(source, defuse=self.defuse, timeout=self.timeout, lazy=False)
  File "/usr/local/lib/python3.8/site-packages/xmlschema/resources.py", line 349, in __init__
    self.source = source
  File "/usr/local/lib/python3.8/site-packages/xmlschema/resources.py", line 366, in __setattr__
    self._root, self._text, self._url = self._fromsource(value)
  File "/usr/local/lib/python3.8/site-packages/xmlschema/resources.py", line 430, in _fromsource
    return self.parse(resource).getroot(), None, url
  File "/usr/local/lib/python3.8/site-packages/xmlschema/resources.py", line 590, in parse
    return ElementTree.parse(source)
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/xml/etree/ElementTree.py", line 1202, in parse
    tree.parse(source, parser)
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/xml/etree/ElementTree.py", line 595, in parse
    self._root = parser._parse_whole(source)
xml.etree.ElementTree.ParseError: not well-formed (invalid token): line 73, column 0
```

**Comment by coderpatros** [comment]
@agentram thanks for the report and repro

**Comment by coderpatros** [comment]
Fix available in version 0.4.3
